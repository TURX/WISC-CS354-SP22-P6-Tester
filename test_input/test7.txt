0 100 31