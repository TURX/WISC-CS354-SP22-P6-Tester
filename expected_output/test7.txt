Reading character starting at index 0 to 100 with an interval of 31

Reading character at index 0   : data = A : hit count = 0   : miss count = 1   : read data count = 1
Reading character at index 31  : data = d : hit count = 1   : miss count = 1   : read data count = 1
Reading character at index 62  : data = N : hit count = 1   : miss count = 2   : read data count = 2
Reading character at index 93  : data =   : hit count = 1   : miss count = 3   : read data count = 3
